mutt_mailbox.c

```diff
@@ -368,7 +368,7 @@ struct Mailbox *mutt_mailbox_next(struct Mailbox *m_cur, struct Buffer *s)
     if (m_res)
       return m_res;
 
-    mutt_mailbox_check(m_cur, MUTT_MAILBOX_CHECK_FORCE); /* mailbox was wrong - resync things */
+    mutt_mailbox_check(m_cur, MUTT_MAILBOX_CHECK_POSTPONED); /* mailbox was wrong - resync things */
```

Comment:
`/* mailbox was wrong - resync things */` I don't really understand this comment. How does checking the postponded number help here?

Reply:
I have no idea - I just changed the enumerator name here.

Reply:
<img width="923" alt="Screenshot 2024-08-12 at 09 38 50" src="https://github.com/user-attachments/assets/b8dcc58b-3ae5-4bed-b88a-1d3cc79861d3">

I'd probably remove it. 😆